autovalet_goal_generation/setup.py

```diff
@@ -0,0 +1,10 @@
+# To install the module for import into main state machine
+
+from distutils.core import setup
+from catkin_pkg.python_setup import generate_distutils_setup
+
+d = generate_distutils_setup(
+    packages=['autovalet_goal_generation'],
+    package_dir={'': 'src'}
+)
+setup(**d)
```

Comment:
why do we need this specifically in goal generation?

Reply:
we need it in general. It's the way to generate python modules which can be imported across ros into packages/nodes anywhere
for eg: we can now do from autovalet_goal_generation.goal_generator import goal_generator in say some node that we wrote in lane detection or the state machine etc.

Reply:
agreed. i already put in in the lane detection pkg in my branch